docs/client/full-api/concepts.html

```diff
@@ -148,6 +148,16 @@ <h2 id="structuringyourapp">Structuring your application</h2>
     `node_modules` directories will not be available to your Meteor code. Use
     `Npm.depends` in your package `package.js` file for that.
 
+- **.***
+
+    Files starting witha  dot (`.`) are not loaded anywhere, including the `.meteor` folder.
+
+The following folders are not loaded anywhere:
+
+- packages (covered later)
+- programs (for legacy reasons)
+- cordova-build-override
```

Comment:
Is [this](https://github.com/meteor/meteor/wiki/Meteor-Cordova-Phonegap-integration#advanced-build-customization) the docs you are talking about?